script.js

```diff
@@ -0,0 +1,37 @@
+const cardArray = [
+  {
```

Comment:
Good idea setting this up as an array to start with - you would have had to do it anyway, but by doing it early you're saving yourself some work next week!

Reply:
Thank you and yes I was thinking the final product when I started planning and thinking about the solution.